src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -18,6 +18,10 @@ public void testIntelDeflaterIsAvailable(){
             throw new SkipException("IntelDeflater not available on this platform");
         }
 
+        if ( SystemUtils.OS_ARCH.equals("ppc64le") ) {
```

Comment:
I think create a new `NativeUtils` class, as I suspect we'll be writing more methods like this in the future.